openlane/steps/odb.py

```diff
@@ -461,6 +463,8 @@ def get_command(self) -> List[str]:
             pin,
             "--port-protect",
             self.config["DIODE_ON_PORTS"],
+            "--threshold",
+            "Infinity",
         ]
 
     def run(self, state_in: State, **kwargs) -> Tuple[ViewsUpdate, MetricsUpdate]:
```

Comment:
GPL_CELL_PADDING warning in this method is no applicable anymore

Reply:
We've discussed privately- it doesn't have to do with the legalization, rather, it has to do with the diode insertion itself